### ISSUE TITLE
[Frontend] Accessibility Issues

### ISSUE THREAD

**Issue by mmtftr** [bounswe/bounswe2024group1#644]
# 🐞 Bug report

## 🏡 Component 
- [x] Frontend
## ✅ Expected Behavior
There should be no accessiblity issues in the frontend

## 📃 Related requirements
[Link to section](https://github.com/bounswe/bounswe2024group1/wiki/Requirements#341-the-web-system-shall-implement-the-wcag-22-standard-to-ensure-accessibility)
> 3.4.1. The web system shall implement the WCAG 2.2 standard to ensure accessibility.

## ❌ Actual Behavior

There are accessibility issues in the tag page, home page and some other pages. These issues include contrast problems and missing aria-labels.

## 🚀 Steps to Reproduce the Problem
1. Go to page in frontend
2. Run Lighthouse

## 💻 Specifications
Production, Accessed using Arc (Chromium-based browser)

## 👁‍🗨 Reviewer
@mmtftr

**Comment by NazireAta** [comment]
closed due to #648

**Comment by NazireAta** [comment]
closed due to #648
